### PR TITLE
initial readme

### PR DESCRIPTION
**Pull request by jodygarnett**
Outline repository purpose, use of copyright and license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+
+## License
+
+All work here is copyright The Open Source Geospatial Foundation, unless otherwise stated it is made available to you under an appropriate [Creative Commons](https://creativecommons.org) license.
+
+Document licenses for software developers:
+
+* [CCBY](https://creativecommons.org/licenses/by/4.0/) - permissive license similar to MIT or BSD allowing content to be remixed by other authors, as long as they give us a credit somewhere. This is a great choice for outreach and increases our brand awareness.
+
+* [CCBYSA](https://creativecommons.org/licenses/by-sa/4.0/) - protective license similar to gpl allows content to be remixed by other authors, as long as what they are writing is also using a share-alike license.
```
Comment (mloskot) [inline README.md]: s/gpl/GPL/ ?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+
+* [CC0](https://creativecommons.org/publicdomain/) - creative commons public domain tools are used for outreach where we want the information to have as far a reach as possible. When using this approach we do not even ask for attribution! We have to be careful using this license to avoid including any content or photos that we did not create ourselves.
+
+Not all of our content is licensed for modification:
+
+* [CCBYND](https://creativecommons.org/licenses/by-nd/4.0/) - some documents are not useful to share in modified form: our organization charter, memorandums of understanding with partners, official legal documents, letters of recommendation, letters of support. In these cases we allow free distribution, but not modification, of said content. Authors may still quote or cite these materials, and this github repository provides a stable URL. Be sure to include the revision when citing as these documents change over time.
```
Comment (mloskot) [inline README.md]: s/github/GitHub/ ? It is a proper name

Comment (jodygarnett) [inline README.md]: Thanks for the review!

Comment (mloskot) [inline README.md]: Taking this as opportunity to learning GitHub code review capacity, not trying to be PITA :-)

**Pull request merged**